### DISCUSSION TITLE
getting problem in play bgmi in bluestack 5

### DISCUSSION THREAD

**Discussion by dilip1241** [community/community#66780] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

getting problem in playing BGMI in bluestack 5 . It is showing that your device setup  is not compitable

**Comment by queenofcorgis** [comment]
Hi @dilip1241 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this like: https://support.bluestacks.com/hc/en-us. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
